internal/distributed/proxy/service.go

```diff
@@ -14,20 +14,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package grpcproxy
+package proxy
```

Comment:
I prefer naming this way while it conflicts other package's naming convention. Not sure we should change that here.

Reply:
This package is quoted only in 2 position, so for compile level it's ok to change here. 
But it's sub package `proxy/client` is still named as `grpcclient` which is widely used. 
So maybe we should not change for now, and leave it for another PR focused to change them all together